templates/common/_base/files/iptables-modules.yaml

```diff
@@ -0,0 +1,6 @@
+mode: 0644
```

Comment:
Youll need to add filesystem here as 4.4 uses ignition spec v2

Reply:
Thanks!

Reply:
Thanks!I think I got the right name...

Reply:
yep!